src/reflow/mod.rs

```diff
@@ -809,4 +827,19 @@ paragraph"#,
             assert_eq!(replacement.as_str(), expected_content);
         }
     }
+
+    #[test]
+    fn reflow_line_delimiters() {
+        let test_data = vec![
+            ("Two lines\nhere", "\n"),
+            ("Two lines\r\nhere", "\r\n"),
+            ("Two lines\n\rhere", "\n\r"),
+            ("Two lines\rhere", "\r"),
+            ("Two lines\nhere\rand more", "\r"),
```

Comment:
I would like this to be `\n` rather than `\r` (I think we should warn about single `\r` being present, it's usually a fragment of some screwed up newline conversion tool, and rarely intentional)